src/test/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/ReportResourceTest.kt

```diff
@@ -1048,6 +1063,7 @@ class ReportResourceTest : SqsIntegrationTestBase() {
           location = "LEI",
           title = "Updated report 11124143",
           description = "Updated incident report of type find of illicit items",
+          duplicatedReportId = Optional.of(UUID.fromString(duplicatedExistingReport.id.toString())),
```

Comment:
out of curiosity, isn’t `duplicatedExistingReport.id` already a UUID? could this be:
```kotlin
Optional.of(duplicatedExistingReport.id)
```

(similar below)

Reply:
we know that once saved the report must have an id so you could
`Optional.of(duplicatedExistingReport.id!!)`